plugins/frontend/backstage-plugin-cloudsmith/src/components/CloudsmithRepositoryAuditLogCard/Content.test.tsx

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2022 Larder Software Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { render } from '@testing-library/react';
+import {
+  AnyApiRef,
+  discoveryApiRef,
+  FetchApi,
+  fetchApiRef,
+  errorApiRef,
+} from '@backstage/core-plugin-api';
+import {
+  setupRequestMockHandlers,
+  TestApiProvider,
+} from '@backstage/test-utils';
+import { setupServer } from 'msw/node';
+import { Content } from './Content';
+
+const mockResponse = {
+  packages: {
```

Comment:
You will need to change the mockResponse here to look like the response from the usage api.

Reply:
You already have an example in the mocks file, maybe you can just import it?